differential Expression Import
==============================

Experiment Information
----------------------

Specify Experiment Title/Description, Organism, and Pubmed ID for
keeping track of uploaded data.

Data information
----------------

What is this?
~~~~~~~~~~~~~

At PATRIC, you can upload your own pre-processed differential expression
datasets generated by microarray, RNA-Seq, or proteomic technologies to
your workspace and analyze them using annotations and analysis tools.
Currently, PATRIC only supports differential gene expression data in the
form of log ratios, generated by comparing samples/conditions/time
points. You may also compare your data with other transcriptomics
datasets available at PATRIC. Data uploaded in your workspace is private
and protected.

Experiment Data
~~~~~~~~~~~~~~~

Upload a data file containing differential gene expression values in the
form of log ratios. The file should be in one of the supported formats
described below. Optionally, you may also upload metadata related to
sample comparisons in the prescribed format to help later in the data
analysis.

Supported IDs
~~~~~~~~~~~~~

1. RefSeq Locus Tag
2. PATRIC Feature ID
3. NCBI GI Number
4. NCBI Protein ID
5. SEED ID
6. PATRIC Legacy ID

File Format
~~~~~~~~~~~

Currently, PATRIC allows you to upload your transcriptomics datasets in
the form of differential gene expression measured as log ratios. Data
can be uploaded in multiple file formats: comma separated values (.csv),
tab delimited values (.txt), or Excel (.xls or .xlsx). Click to download
`Sample Data template <../_static/Sample-Genes.xlsx>`__ in Gene Matrix Format.
Files should contain data in one of the following formats:

1. Gene Matrix:

   +-----------+---------------+---------------+
   | Gene ID   | Comparison1   | Comparison2   |
   +===========+===============+===============+
   | b0002     | 0.767         | -1.316        |
   +-----------+---------------+---------------+
   | b0003     | 0.815         | -1.841        |
   +-----------+---------------+---------------+
   | b0004     | 0.856         | -1.643        |
   +-----------+---------------+---------------+

2. Gene List:

   Data is presented in three columns: Gene ID, Sample ID, and
   expression value. Expression value should be in the form of log ratio
   (i.e. log2 (test/control)). Â Below is an example of transcriptomics
   data in Gene List format:

   +-----------+-----------------+-------------+
   | Gene ID   | Comparison ID   | Log Ratio   |
   +===========+=================+=============+
   | b0002     | Comparison1     | 0.767       |
   +-----------+-----------------+-------------+
   | b0003     | Comparison1     | 0.815       |
   +-----------+-----------------+-------------+
   | b0002     | Comparison2     | -1.316      |
   +-----------+-----------------+-------------+

Experiment Type
~~~~~~~~~~~~~~~

This field specifies the the experiment type Transcriptomics,
Proteomics, or Phenomics.

ID Type
~~~~~~~

In order to take full advantage of PATRIC data, gene ID's provided in
the experiment data are mapped to PATRIC. Due to differences in
annotation that may exist some genes may go unmapped. Unmapped genes
will be excluded from subsequent analysis.

File Type
~~~~~~~~~

This field specifies whether the uploaded Experiment Data File is
tab/comma delimited or an excel file.

Optional Metadata
-----------------

PATRIC allows you to upload an additional file containing key metadata
attributes about the comparisons. This will enhance your analysis when
using the Heatmap and clustering tools. The metadata table can be
uploaded as a tab/comma delimited or Excel file. Note: The column names
should be the same as in the example template. Also, comparison IDs in
the metadata file should match those in the data file. Click to download
`Metadata Template <../_static/Sample-Metadata.xlsx>`__.
